apps/dialtone-documentation/docs/about/whats-new/posts/2024-4-11.md

```diff
@@ -0,0 +1,282 @@
+![dialtone vue bundle size after, without tree shaking](https://private-user-images.githubusercontent.com/87546543/314394232-e11b209c-d920-48e5-b1c5-238c4cdf899a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI4NTM2MjEsIm5iZiI6MTcxMjg1MzMyMSwicGF0aCI6Ii84NzU0NjU0My8zMTQzOTQyMzItZTExYjIwOWMtZDkyMC00OGU1LWIxYzUtMjM4YzRjZGY4OTlhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDExVDE2MzUyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmZDA5OTczNDliMGQ2NTc3YWY5N2JlZjM1ZjQ4YjhlN2YzYzk3ZWEzOTJhYmY2YzZkNGEzNDIyYjJiOTY5YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MroDR_6OVbUqKxfPxl_-FztWejwFkihCJWlwmbP30oQ)
+
+### After (with tree shaking)
+
+![dialtone vue bundle size after, with tree shaking](/assets/images/dialtone-vue-bundle-size-after-tree-shaking.png)
+
+```bash
+✓ 281 modules transformed.
+dist/style.css                                    34.04 kB │ gzip: 5.80 kB
+dist/chunks/dropdown_constants-EUcDxBrX.js         0.14 kB │ gzip: 0.15 kB │ map:  0.41 kB
+dist/chunks/notice_constants-c--hBFQw.js           0.18 kB │ gzip: 0.16 kB │ map:  0.46 kB
+dist/chunks/list_item_constants-LTUc74pD.js        0.21 kB │ gzip: 0.19 kB │ map:  0.60 kB
+dist/chunks/stack_constants-u7tNqGtc.js            0.32 kB │ gzip: 0.21 kB │ map:  0.85 kB
+dist/lib/emoji.js                                  0.35 kB │ gzip: 0.24 kB │ map:  0.10 kB
+dist/chunks/icon_constants-OpYAAKwF.js             0.37 kB │ gzip: 0.21 kB │ map:  0.78 kB
+dist/lib/validators.js                             0.38 kB │ gzip: 0.28 kB │ map:  0.92 kB
+dist/chunks/link_constants-vIUB92L4.js             0.42 kB │ gzip: 0.25 kB │ map:  1.10 kB
+dist/lib/codeblock.js                              0.52 kB │ gzip: 0.36 kB │ map:  0.54 kB
+dist/lib/mixins.js                                 0.57 kB │ gzip: 0.31 kB │ map:  0.10 kB
+dist/lib/combobox.js                               0.60 kB │ gzip: 0.32 kB │ map:  0.11 kB
+dist/lib/button-group.js                           0.91 kB │ gzip: 0.53 kB │ map:  1.44 kB
+dist/lib/time-pill.js                              0.94 kB │ gzip: 0.53 kB │ map:  1.81 kB
+dist/lib/constants.js                              1.05 kB │ gzip: 0.49 kB │ map:  2.55 kB
+dist/chunks/_plugin-vue2_normalizer-u6G_3nkj.js    1.11 kB │ gzip: 0.58 kB │ map:  0.15 kB
+dist/lib/dropdown.js                               1.26 kB │ gzip: 0.62 kB │ map:  0.69 kB
+dist/lib/dates.js                                  1.34 kB │ gzip: 0.64 kB │ map:  6.00 kB
+dist/lib/icon.js                                   1.36 kB │ gzip: 0.71 kB │ map:  1.95 kB
+dist/lib/link.js                                   1.37 kB │ gzip: 0.69 kB │ map:  2.06 kB
```

Comment:
A bit much to show this whole entire list. Maybe just show a few examples to show they are split into components now and do `...`